### DISCUSSION TITLE
Minimization of maximum distance for a vehicle depends on given dimension capacity

### DISCUSSION THREAD

**Discussion by gregy4** [google/or-tools#3105] [Routing (and legacy CP) questions]
Hello,
  I tried to model simple example with 25 nodes and 3 vehicles. Based on given capacity of the dimension I get different results.

  With capacity=1000, I get maximum distance 237. With capacity=200 only 164.

  Distances in this example are not real distances but processing times at nodes that doesn't depend on target node.

  Why vrp solver doesn't minimize vehicle distance when capacity=1000 and finishes too early ?  Is it because distances aren't real distances (in this case I need alternative to define processing time at a node) ?  

Thanks,
Jan
[Example.zip](https://github.com/google/or-tools/files/7963546/Example.zip)

**Comment by Mizux** [comment]
in you example you didn't enable the guided local search...

```diff
        // Setting first solution heuristic.
        RoutingSearchParameters searchParameters =
                main.defaultRoutingSearchParameters()
                        .toBuilder()
                        .setTimeLimit(Duration.newBuilder().setSeconds(10000).build())
                        .setLogSearch(true)
                        .setFirstSolutionStrategy(FirstSolutionStrategy.Value.PATH_CHEAPEST_ARC)
+                       .setLocalSearchMetaheuristic(LocalSearchMetaheuristic.Value.GUIDED_LOCAL_SEARCH)
                        .build();
```
